security/access_control.rst

```diff
@@ -65,63 +65,34 @@ Take the following ``access_control`` entries as an example:
     .. code-block:: php
 
         // config/packages/security.php
-        use Symfony\Component\DependencyInjection\ContainerBuilder;
-        use Symfony\Config\SecurityConfig;
-
-        return static function (ContainerBuilder $container, SecurityConfig $security): void {
-            $container->setParameter('env(TRUSTED_IPS)', '10.0.0.1, 10.0.0.2');
-            // ...
-
-            $security->accessControl()
-                ->path('^/admin')
-                ->roles(['ROLE_USER_PORT'])
-                ->ips(['127.0.0.1'])
-                ->port(8080)
-            ;
-            $security->accessControl()
-                ->path('^/admin')
-                ->roles(['ROLE_USER_IP'])
-                ->ips(['127.0.0.1'])
-            ;
-            $security->accessControl()
-                ->path('^/admin')
-                ->roles(['ROLE_USER_HOST'])
-                ->host('symfony\.com$')
-            ;
-            $security->accessControl()
-                ->path('^/admin')
-                ->roles(['ROLE_USER_METHOD'])
-                ->methods(['POST', 'PUT'])
-            ;
-            // ips can be comma-separated, which is especially useful when using env variables
-            $security->accessControl()
-                ->path('^/admin')
-                ->roles(['ROLE_USER_IP'])
-                ->ips([env('TRUSTED_IPS')])
-            ;
-            $security->accessControl()
-                ->path('^/admin')
-                ->roles(['ROLE_USER_IP'])
-                ->ips(['127.0.0.1', '::1', env('TRUSTED_IPS')])
-            ;
-
-            // for custom matching needs, use a request matcher service
-            $security->accessControl()
-                ->roles(['ROLE_USER'])
-                ->requestMatcher('App\Security\RequestMatcher\MyRequestMatcher')
-            ;
-
-            // require ROLE_ADMIN for 'admin' route. You can use the shortcut route('xxx') method,
-            // instead of attributes(['_route' => 'xxx']) method
-            $security->accessControl()
-                ->roles(['ROLE_ADMIN'])
-                ->attributes(['_route' => 'admin'])
-            ;
-            $security->accessControl()
-                ->roles(['ROLE_ADMIN'])
-                ->route('admin')
-            ;
-        };
+        namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+        use App\Security\RequestMatcher\MyRequestMatcher;
+
+        return App::config([
+            'parameters' => [
+                'env(TRUSTED_IPS)' => '10.0.0.1, 10.0.0.2',
```

Comment:
Noted we cannot use `env('TRUSTED_IPS')` here; as surrounding `%...%` would be added.